design-tokens/tokens/component.default.json

```diff
@@ -5858,6 +5858,291 @@
     }
   },
   "dataCell": {
+    "primary": {
+      "paddingX": {
+        "$type": "number",
+        "$value": "{base.static.spacing.small}",
```

Comment:
To make it easier for things to stay in sync in future, can we have these all reference `{dataCell.paddingX}` or whatever the property is instead?

Then just for the ones that deviate (like fontWeight) have it be its own value?

Reply:
Yeah I'll make that change. Also I think the `dataCell.fontWeight` and `dataCell.enabled.fontColor` are wrong. I wasn't sure if I should touch those in this PR though since they don't fall under the scope of `dataCell.primary`

Reply:
Thanks for flagging, feel free to fix them thank you!

Reply:
Fixed